data/sd/legislature/Lynn-V-Schneider-3546dbf4-9885-4e13-a2a3-05f201932adf.yml

```diff
@@ -1,22 +0,0 @@
-id: ocd-person/3546dbf4-9885-4e13-a2a3-05f201932adf
-name: Lynn V. Schneider
-given_name: Lynn
-family_name: Schneider
-email: Lynn.Schneider@sdlegislature.gov
-image: https://lawmakerdocuments.blob.core.usgovcloudapi.net/photos/schneider, lynn
-  rep-2458 full rt.jpg
```

Comment:
@sroomf i believe this file should be un-deleted as it is their original file.

Reply:
oh noooo i think i've deleted so many incorrect files :/ linting made two files that looked the same so I guessed on which one should be deleted